### ISSUE TITLE
Retrain with FLAN-T5-base

### ISSUE THREAD

**Issue by Bachstelze** [PrithivirajDamodaran/Gramformer#37]
We evaluated your and various T5-models on syntax: [Evaluating PaLM-FLAN-T5 and previous models on syntax](https://github.com/Bachstelze/clams/blob/master/combined_Evaluating_PaLM_Flan_T5_on_Syntax_20_03.pdf)

Gramformer is the the grammar-tuned dot:
![figure-contrast](https://user-images.githubusercontent.com/19904888/226266492-214422e9-b47f-4139-9e3b-dd5a1fed7b60.jpg)

The evaluation suggests using [FLAN-T5-base](https://huggingface.co/google/flan-t5-base)

**Comment by akesh1235** [comment]
@Bachstelze  Yeah, Mr. Peter Szemraj trained using flan-t5-large model, size of the model is 3.13 GB as you evaluated sir.
Any suggetion, how can this flan-t5-large model from Mr. Peter Szemraj can be improved further.

**Comment by Bachstelze** [comment]
@akesh1235 you probably mean [flan-t5-large-grammar-synthesis](https://huggingface.co/pszemraj/flan-t5-large-grammar-synthesis). This model is only trained on the [JFLEG](https://paperswithcode.com/dataset/jfleg) dataset. You can add more datasets for the english-centric Flan-T5:
* https://paperswithcode.com/datasets?task=grammatical-error-correction
* https://huggingface.co/datasets?other=grammatical-error-correction
Also, you can add more languages: https://github.com/PrithivirajDamodaran/Gramformer/issues/32
Flan-T5 is already pre-trained in plenty languages: English, Spanish, Japanese, Persian, Hindi, French, Chinese, Bengali, Gujarati, German, Telugu, Italian, Arabic, Polish, Tamil, Marathi, Malayalam, Oriya, Panjabi, Portuguese, Urdu, Galician, Hebrew, Korean, Catalan, Thai, Dutch, Indonesian, Vietnamese, Bulgarian, Filipino, Central Khmer, Lao, Turkish, Russian, Croatian, Swedish, Yoruba, Kurdish, Burmese, Malay, Czech, Finnish, Somali, Tagalog, Swahili, Sinhala, Kannada, Zhuang, Igbo, Xhosa, Romanian, Haitian, Estonian, Slovak, Lithuanian, Greek, Nepali, Assamese, Norwegian

**Comment by ILG2021** [comment]
I have fine tune a flan-t5 model on my own small dataset about 1500 rows. But i find it not works. I only want to correct the error in my own dataset. others just to return the original text. it is not run as i expect.
